api/v1/controllers/collectors.js

```diff
@@ -109,15 +109,9 @@ function attachTemplate(sg) {
  */
 function postCollector(req, res, next) {
   const collectorToPost = req.swagger.params.queryBody.value;
-  const resultObj = { reqStartTime: req.timestamp };
   const toPost = req.swagger.params.queryBody.value;
   helper.model.create(toPost)
   .then((o) => {
-    if (helper.loggingEnabled) {
```

Comment:
why are you removing this?

Reply:
The postCollector is just a stub for your heartbeat test to run for now and this block is not required. Also, this was checked in with defining the `utils` variable below.